lib/simctl.js

```diff
@@ -157,12 +157,13 @@ async function getDevices (forSdk:string = null):Object {
       //   iPhone 4s
       //   A99FFFC3-8E19-4DCF-B585-7D9D46B4C16E
       //   Shutdown
-      let lineRe:RegExp = /^    ([^\(]+) \(([^\)]+)\) \(([^\)]+)\)/;
+      let lineRe:RegExp = /([^\s].+) \((\w+-.+\w+)\) \((\w+)\)/;
       let lineMatch:Object = lineRe.exec(line);
       if (lineMatch === null) {
-        throw new Error('Could not match line');
+        throw new Error('Could not match line: ' + line);
```

Comment:
I've been adding a comment including a saved regexp test, with the regexp and a variety of examples of the different sorts of input text expected from different versions, for clarity. See, for instance, https://github.com/appium/appium-adb/blob/71643c324927a3bedf340937d189437c398271ff/lib/tools/apk-utils.js#L90